code/datums/mind.dm

```diff
@@ -75,12 +75,15 @@
 	src.key = key
 
 /datum/mind/proc/transfer_to(mob/new_character)
+	if(!new_character)
```

Comment:
silently failing here is questionable, why would you ever call `transfer_to(null)`?